utils/gear-runtime-test-cli/src/util.rs

```diff
@@ -225,9 +215,40 @@ pub(crate) mod gear {
         SessionKeys, System,
     };
 
-    utils!();
+    utils! {
+        {authorities, t},
+        {
+            pub fn authority_keys_from_seed(s: &str) -> (AccountId32, BabeId, GrandpaId) {
+                (
+                    get_account_id_from_seed::<sr25519::Public>(s),
+                    get_from_seed::<BabeId>(s),
+                    get_from_seed::<GrandpaId>(s),
+                )
+            }
+        },
+        {
+            SessionConfig {
+                keys: authorities
+                    .iter()
+                    .map(|x| {
+                        (
+                            x.0.clone(),
+                            x.0.clone(),
+                            SessionKeys {
+                                babe: x.1.clone(),
+                                grandpa: x.2.clone(),
+                            },
+                        )
+                    })
+                    .collect(),
+            }
+            .assimilate_storage(&mut t)
+            .unwrap();
```

Comment:
Isn't `expect` better than `unwrap()`?

Reply:
I don't think it matters here: we use `unwrap` in every single place when constructing storage for test externalities - why suddenly make this an exception?